monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Para experimentar este poder, necesitarás elegir un monedero. El primer aspecto crítico de cualquier monedero es su confiabilidad. Sólo monederos recomendados por la comunidad (y el sitio [getmonero.org](https://getmonero.org/es/)) son mencionados en este artículo. A continuación, querrás elegir un monedero que se ejecute en tu dispositivo de cómputo preferido — iPhone, Android, ordenador Portátil o de Escritorio. Finalmente, deberías elegir un monedero que concuerde con tus necesidades de privacidad. Este artículo te guiará en ello.
+
+Algunos monederos operan usando su propia copia de la blockchain de Monero. A esto también se le llama ejecutar un nodo completo. Te da control sobre la puntualidad de tus transacciones. Aunque puedes confiar en un nodo remoto sin el riesgo añadido de perder tus Monero, si dependes en la disponibilidad del nodo remoto — siempre podrás recibir, pero es posible que no seas capaz de enviar. Otro beneficio de ejecutar tu propio nodo completo es el orgullo de contribuir a la red de Monero. Nodos completos honestos hacen más fuerte a Monero. Aprende más aquí: [monero.how/how-to-run-monero-node (en inglés)](https://www.monero.how/how-to-run-monero-node).
+
+Los beneficios de confiar, en su lugar, en nodos remotos — usando lo que se conoce como monederos ligeros, o algunas veces livianos — incluyen que no necesitas recursos de almacenamiento para la blockchain de Monero. La (no reducible) blockchain es más grande que 50 GB y en aumento. Algunos monederos que utilizan nodos remotos completos también usan menos poder de procesamiento y banda ancha de red. Si estos recursos son limitados en tu dispositivo de computación, un monedero ligero puede ser preferido. Este casi siempre será el caso para un monedero ejecutándose en un teléfono inteligente.
+
+Los monederos ligeros vienen en dos tipos: de protección de claves-de-visualización y de compartir claves-de-visualización. Los monederos ligeros de protección de claves de visualización mantienen tus claves de visualización completamente privadas, y por consecuencia tienen que descargar la cadena de bloques, o al menos la porción de ella después de la fecha de creación del monedero para calcular el balance del Monero en el monedero. Por otro lado, los monederos ligeros de compartir claves de visualización comparten la clave de visualización con el servidor ejecutando el nodo remoto para calcular el balance de entrada del monedero en el servidor. Estos eficientes monederos son a veces llamados aplicaciónes de monedero. El compartir las claves de visualización reduce la privacidad si el servidor ejecutando el nodo de reconocimiento de clave de visualización no es confiable, debido a que permite la observación de transacciones de entrada. Nótese, sin embargo, que algunas veces los dueños de monederos comparten sus claves de visualización para auditoría de todas formas — no da lugar para transferir Monero.
```

Comment:
Nótese, sin embargo, que algunas veces los dueños de monederos comparten sus claves de visualización para auditoría de todas formas — **esto**, no da lugar para transferir Monero.

*Like we are adding a "—", this represents a "separate" sentence, so we need to make clear that we are talking about what key holders do by sharing their private veiw keys that does not allow transfer just by sharing. Not sure if you understand my point.